### DISCUSSION TITLE
Can snapcraft register a program as a service and still access it from the command line?

### DISCUSSION THREAD

**Discussion by IoTServ** [goreleaser/goreleaser#1798] [Q&A] [answered]
When goreleaser packages a golang program as a snapcraft program, does goreleaser not put the binary files of the program in the path directory for the command line to use after registering the program as a service? I found that snapcraft cannot be accessed from the command line after snapcraft is set to simple

**Comment by caarlos0** [comment, marked as answer]
This question seems like more of a snapcraft than a goreleaser question... maybe try their forum?

**Reply by IoTServ** [reply]
> This question seems like more of a snapcraft than a goreleaser question... maybe try their forum?

ok，I will try.thanks
